components/bank/forms/ibcSendForm.tsx

```
@@ -1,598 +1,187 @@
+          s.shadowRoot?.querySelector('div[open]')
+        );
+
+        const isSkipGoDialogOpened = allInputs.filter(Boolean).length > 0;
+
+        if (isSkipGoDialogOpened) {
+          // Get all the buttons and other interactive elements inside the modal
+          const modalContent = document.querySelector('wcm-modal');
+
+          // Check if the click was directly on the backdrop and not on any content
+          const isClickOnContent =
+            (modalContent && modalContent.contains(event.target as Node)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)